nvflare/apis/utils/job_utils.py

```diff
@@ -81,3 +81,9 @@ def convert_legacy_zipped_app_to_job(zip_data: bytes) -> bytes:
                     out_zip.writestr(info, content)
 
         return writer.getvalue()
+
+
+def load_job_def(from_path: str, def_name: str) -> bytes:
```

Comment:
This is duplicate with get_app() in the job_def_manager.py. We should consolidate them with a single one.

Reply:
Agreed. @yhwen Could you advise how?